app/src/main/java/com/antosik/benchproject/app/popular/movies/viewModel/PopularMoviesViewModel.kt

```diff
@@ -37,14 +40,21 @@ class PopularMoviesViewModel @Inject constructor(
         popularMoviesFragmentNavigator.navigateToMovieDetailsFragment(movieId)
     }
 
+    fun refreshPopularMovie() {
```

Comment:
Should be rather `onRefreshPopularMovies` as it is called from outside of VM.

Reply:
I changed it